### ISSUE TITLE
Affichage d'options d'édition pour des éditeurs d'autres groupes

### ISSUE THREAD

**Issue by vbombaerts** [SPW-DIG/metawal-core-geonetwork#944]
Je viens de créer un nouveau groupe (CRAW) dans lequel j'ai deux éditeurs et un utilisateur enregistré.
Quand je me loggue avec un éditeur de ce groupe, dans l'interface d'édition, certaines fiches apparaissent comme éditables (elles ne le sont pas). J'ai aussi l'impression d'avoir le droit de changer les privilèges pour ces fiches. Est-il possible de rectifier ces artefacts ?

![image](https://github.com/user-attachments/assets/23652b2c-7e65-4f9e-acb2-2c8918087b49)
![image](https://github.com/user-attachments/assets/cf628bb6-ca27-41d1-9454-430d750975b4)

**Comment by fxprunayre** [comment]
En effet, ça semble lié au fait que la fiche est marqué comme `editable` pour le groupe `1` (qui est internet)

![image](https://github.com/user-attachments/assets/78a05fcf-09b2-4520-8005-8cff657182a6)

Quand on ouvre l'éditeur, il y a une autre vérification qui fait qu'on ne peut pas éditer la fiche (car la notion de `editable` pour group internet est ignorée).

En changeant les privilèges avec
![image](https://github.com/user-attachments/assets/4c466d48-2b50-4881-a412-a42ac14b9345)

ou avec dépublier/publier
![image](https://github.com/user-attachments/assets/585b8b17-0d83-4320-ab05-6539732ff612)

la notion `editable` pour le group internet n'est pas ajoutée (cf. https://github.com/geonetwork/core-geonetwork/blob/main/services/src/main/java/org/fao/geonet/api/records/MetadataSharingApi.java#L572-L575).

Donc soit on a des changements fait via l'API ou une autre action dans l'application?

J'aurai tendance à dire qu'on peut enlever ces infos:

```sql
SELECT * FROM operationallowed
WHERE operationid = (SELECT id FROM operations WHERE name = 'editing')
AND groupid = (SELECT id FROM groups WHERE name = 'all')
```

mais ce serait bien de trouver comment elles sont ajoutées.

**Comment by vbombaerts** [comment]
Oui, il me semble qu'on peut supprimer cette info. Les groupes internet et intranet sont des groupes pour lesquels on ne gère pas les utilisateurs et donc des droits d'édition ne peuvent pas être associés à ces groupes.
